doc/introduction/introducing_zephyr.rst

```diff
@@ -31,35 +32,49 @@ Distinguishing Features
 The Zephyr kernel offers a number of features that distinguish it from other
 small-footprint OSes:
 
-#. **Single address-space**. Combines application-specific code
+**Single address-space**
+   Combines application-specific code
    with a custom kernel to create a monolithic image that gets loaded
    and executed on a system's hardware. Both the application code and
    kernel code execute in a single shared address space.
 
-#. **Highly configurable**. Allows an application to incorporate *only*
+**Highly configurable / Modular for flexibility**
+   Allows an application to incorporate *only*
    the capabilities it needs as it needs them, and to specify their
    quantity and size.
 
-#. **Compile-time resource definition**. Allows system resources
+**Cross Architecture**
+   Supports a wide variety of :ref:`supported boards<boards>` with different CPU
+   architectures and developer tools. Contributions have added support
+   for an increasing number of SoCs, platforms, and drivers.
+
+**Compile-time resource definition**
+   Allows system resources
    to be defined at compile-time, which reduces code size and
-   increases performance.
+   increases performance for resource-limited systems.
 
-#. **Minimal error checking**. Provides minimal runtime error checking
+**Minimal error checking**
+   Provides minimal runtime error checking
    to reduce code size and increase performance. An optional error-checking
    infrastructure is provided to assist in debugging during application
    development.
 
-#. **Extensive suite of services**. Offers a number of familiar services
-   for development:
+**Extensive suite of services**
+   Offers a number of familiar services for development:
 
-   * *Multi-threading Services* for priority-based, non-preemptive and
-     preemptive threads with optional round robin time-slicing.
+   * *Multi-threading Services* for cooperative, priority-based,
+     non-preemptive, and preemptive threads with optional round robin
+     time-slicing.  Includes pthreads compatible API and openThread
```

Comment:
openThread has nothing to do with multi-threading. It is a network protocol and belongs below in the networking section.

Reply:
doh.  Fixed.